14-01-date-input/14-1-EN-date-input.sbv

```diff
@@ -468,13 +461,13 @@ that means everything.
 Or we can use "all" instead of "now"
 
 0:06:36.560,0:06:38.660
-as its a historical pseudoname for "now"
+as it's a historical pseudonym for "now"
 
 0:06:38.820,0:06:41.220
-which is only working for the expiration dates.
+which is used only with expiration dates.
```

Comment:
Why so? Why not "which is used with expiration dates only"?

Reply:
work for
sounds like a job a human who work for his boss
the meaning is about usage, i guess.

Reply:
Hmm, thats interesting, I have the following example in mind:
Somebody has a question, like how to fix some concurrency issue in the app. The other guy gives a suggestion, just use the lock and do this and that( something, gives a solution ). Then he says:" Will this work for you?". Doesnt mean the code is going to be hired for a job, right?
Therefore I assumed one can use the same in this case as well. But Im not sure, what do you think? Or "thats not gonna work" - same, you cant use it.

Reply:
i doubt they work for expiration dates, this is my only point

Reply:
i change my mind, you're right. would you like to avoid the gerund anyway?
"which works for"  instead working?!

Reply:
Yes, sure. Can make all the edits we've discussed so far( the dates precision - anyway you like it ) ? So I can close the PR.

Reply:
Done.

Reply:
Done.